charts/aws-pca-issuer/values.yaml

```diff
@@ -5,6 +5,7 @@ replicaCount: 1
 image:
   repository: public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer
   pullPolicy: IfNotPresent
+  tag: ""
```

Comment:
Do we need to add this here? In our Makefile, we're using this: https://github.com/cert-manager/aws-privateca-issuer/blob/main/Makefile#L282

Reply:
This has no effect, it is only to document the option for users.     
Here an example from the Prometheus community. https://github.com/prometheus-community/helm-charts/blob/main/charts/alertmanager/values.yaml#L11